electrum/subscribe.go

```diff
@@ -58,6 +58,37 @@ func (s *Client) SubscribeHeaders(ctx context.Context) (<-chan *SubscribeHeaders
 	return respChan, nil
 }
 
+// SubscribeHeadersSingle subscribes to receive the header of the current
+// blockchain tip. Unlike SubscribeHeaders, this method only returns the tip
+// and does not listen for new block headers.
+//
+// Worth noting that this action still creates a new subscription in the Electrum
+// server. The protocol does neither support a single-shot request for the
+// current blockchain tip nor subscription cancellation. Although this limitation
+// causes a slight resource overhead on the client, it does not cause a memory
```

Comment:
Perhaps we could mention in the docstring of `SubscribeHeaders` that it may cause a resource leak?

Reply:
Good point! Done!